### PR TITLE
Added Multi-PoW block distribution reports

### PR DESCRIPTION
**Pull request by hansieodendaal**
New content based on simulation code

### CODE REVIEW

File: scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Multi-PoW Mining Simulation Approach
+
+(_The code base is available on GitHub at [**tari-labs/modelling/scenarios/multi_pow_01**](https://github.com/tari-labs/modelling/tree/master/scenarios/multi_pow_01)_)
+
+## Mining
```
Comment (stringhandler) [inline scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md:39]: Maybe add a section/abstract describing the problem and the results to give the reader an idea of the purpose of this document

File: scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+To simulate a constant probability of producing a block at the correct difficulty, an oracle is used that asks each algorithm to produce the next block. Each algorithm runs independently, without any inter-dependencies, using its own difficulty adjustment. Mining is based on the average time it takes to solve a block for a given hash rate and target difficulty (see _**Block Time Estimate**_).
+
+```
+solve_time = (target_difficulty/hash_rate) * gradient_r + self.intercept
+achieved_difficulty = ((solve_time - self.intercept) / self.gradient) * hash_rate
+
+```
+
+If an algorithm did not produce the previous solution, an overhead of 1% of the target solve time is added to the solve time to simulate the inefficiencies of block propagation delay on the network and constructing a new block to solve.
```
Comment (stringhandler) [inline scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md]: As discussed, I think what these inefficiencies include and the chosen value of 1% should be substantiated

File: scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# Multi-PoW Mining Simulation Approach
+
+(_The code base is available on GitHub at [**tari-labs/modelling/scenarios/multi_pow_01**](https://github.com/tari-labs/modelling/tree/master/scenarios/multi_pow_01)_)
+
+## Mining
+
+To simulate a constant probability of producing a block at the correct difficulty, an oracle is used that asks each algorithm to produce the next block. Each algorithm runs independently, without any inter-dependencies, using its own difficulty adjustment. Mining is based on the average time it takes to solve a block for a given hash rate and target difficulty (see _**Block Time Estimate**_).
```
Comment (stringhandler) [inline scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md]: I think someone may ask "why did you not just use the probabilities themselves to determine the next block?"
e.g. you could have had a loop every second and determined the probability for each algo to find a block

File: scenarios/multi_pow_01/reports/Multi-PoW block distribution - (Simulation 01).md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+The simulation for target times of 336.0 s and 144.0 s for algo 1 and algo 2 respectively, corresponding to a `df = [0.40]` adjustment factor, is shown here.
+
+![](https://codimd.s3.shivering-isles.com/demo/uploads/upload_180de2a2e7ad4dfc00b56013be39858b.png)
+
+It is noticeable that, for this nominal case, after the initial difficulty adjustment settling period, delta solve times settles on constant values. Integrated system block time shows a constant average, as shown below.
+
+![](https://codimd.s3.shivering-isles.com/demo/uploads/upload_e60dba90a74229cfdeaa8b38db7a3f73.png)
+
+### Block distrubution (Case 01):
```
Comment (stringhandler) [inline scenarios/multi_pow_01/reports/Multi-PoW block distribution - (Simulation 01).md]: spelling

File: scenarios/multi_pow_01/reports/Multi-PoW block distribution - (Simulation 01).md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+The simulation for target times of 336.0 s and 144.0 s for algo 1 and algo 2 respectively, corresponding to a `df = [0.40]` adjustment factor, is shown here.
+
+![](https://codimd.s3.shivering-isles.com/demo/uploads/upload_2bf4b0cbfa55325c694d2a20ce4d7576.png)
+
+It is noticeable that, for this case with added randomess, delta solve times are random, but its average remain within a constant band. Integrated system block time shows a fairly constant average, as shown below.
+
+![](https://codimd.s3.shivering-isles.com/demo/uploads/upload_ad24f227407730a4627dc7109e04aa72.png)
+
+### Block distrubution (Case 02):
```
Comment (stringhandler) [inline scenarios/multi_pow_01/reports/Multi-PoW block distribution - (Simulation 01).md]: spelling

File: scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md (hunk 1 above)
Comment (hansieodendaal) [reply scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md:39]: Done

File: scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md (hunk 3 above)
Comment (hansieodendaal) [reply scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md]: Done

File: scenarios/multi_pow_01/reports/Multi-PoW block distribution - (Simulation 01).md (hunk 4 above)
Comment (hansieodendaal) [reply scenarios/multi_pow_01/reports/Multi-PoW block distribution - (Simulation 01).md]: Done

File: scenarios/multi_pow_01/reports/Multi-PoW block distribution - (Simulation 01).md (hunk 5 above)
Comment (hansieodendaal) [reply scenarios/multi_pow_01/reports/Multi-PoW block distribution - (Simulation 01).md]: Done

File: scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md (hunk 2 above)
Comment (hansieodendaal) [reply scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md]: Done

**Pull request merged**